docker/bfe-config.js

```diff
@@ -0,0 +1,487 @@
+                    bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+                }
+            });
+
+        }
+
+        var config = {
+/*            "logging": {
+                "level": "DEBUG",
+                "toConsole": true
+            },*/
+            "url" : "http://editor:8000",
+            "baseURI": "http://ld4p.org/",
+            "basedbURI": "http://ldp4p.org:8000",
+            "resourceURI": "http://ldp4p.org/resources",
+            "profiles": [
+		        "static/profiles/bibframe/BIBFRAME 2.0 Agents.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Agents Contribution.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Agents Primary Contribution.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Form.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Language.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 LCC.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Notated Music.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Place.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Publication, Distribution, Manufacturer Activity.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Related Works and Expressions.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Topic.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Serial.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Monograph.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Admin Metadata.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Cartographic.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Sound Recording: Analog.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Sound Recording: Audio CD.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Sound Recording: Audio CD-R.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Moving Image: BluRay DVD.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Moving Image: 35mm Feature Film.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Prints and Photographs.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 RWO.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Title Information.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Edition Information.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 Series Information.json",
+			    "static/profiles/bibframe/BIBFRAME 2.0 DDC.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Item.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Identifiers.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Note.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Rare Materials.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Sound Recording: Audio CD-R.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Load.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 IBC.json"
+   		    ],
+            "startingPoints": [
```

Comment:
We'd also need to edit/simplify the startingPoints menu items, if the above assumptions are true.